shared/chat/inbox/container.js

```diff
@@ -28,7 +34,12 @@ const passesParticipantFilter = (participants: I.List<string>, filter: string, y
   if (!filter) {
     return true
   }
-  const names = participants.filter(p => p !== you).toArray()
+
+  // don't filter you out if its just a convo with you!
+  const justYou = participants.count() === 1 && participants.first() === you
```

Comment:
Seems simpler to do:
```
const justYou = ...
const names = justYou ? participants : participants.filter(p => p !== you)
```
(no need to convert to array either, since we're just calling `some` immediately)